tests/opensource-suites/dash/selectors/groupButtonActions.test.ts

```diff
@@ -0,0 +1,261 @@
+
+import {ControlQA, DialogGroupControlQa, Feature} from '../../../../src/shared';
+
+import DashboardPage from '../../../page-objects/dashboard/DashboardPage';
+import datalensTest from '../../../utils/playwright/globalTestDefinition';
+import {getUrlStateParam} from '../../../suites/dash/helpers';
+import {isEnabledFeature, openTestPage, slct} from '../../../utils';
+
+const PARAMS = {
+    FIRST_CONTROL: {
+        controlTitle: 'test-control',
+        controlItems: ['91000', '98800'],
+        controlFieldName: 'test-control-field',
+        defaultValue: '98800',
+    },
+    SECOND_CONTROL: {
+        controlTitle: 'test-control-2',
+        controlItems: ['1', '2'],
+        controlFieldName: 'test-control-field-2',
+        defaultValue: '2',
+    },
+};
+
+const STATE_CHANGING_TIMEOUT = 1000;
+
+const getNewStateHash = async (page: Page, action: () => Promise<void>) => {
+    const requestPromise = page.waitForRequest('/gateway/root/us/createDashState');
+    await action();
+    const finishedPromise = await requestPromise;
+
+    // check new state
+    const responseState = await finishedPromise.response();
+    const jsonState = await responseState?.json();
+
+    return jsonState?.hash;
+};
+
+datalensTest.describe('Dashboards - Action buttons in group selectors', () => {
+    let skipAfterEach = false;
+
+    datalensTest.beforeEach(async ({page}: {page: Page}) => {
+        // some page need to be loaded so we can get data of feature flag from DL var
+        await openTestPage(page, '/');
+
+        const isEnabledGroupControls = await isEnabledFeature(page, Feature.GroupControls);
+
+        if (!isEnabledGroupControls) {
+            skipAfterEach = true;
+            datalensTest.skip();
+        }
+        const dashboardPage = new DashboardPage({page});
+
+        await dashboardPage.createDashboard({
+            editDash: async () => {
+                await dashboardPage.addSelectorsGroup([
+                    PARAMS.FIRST_CONTROL,
+                    PARAMS.SECOND_CONTROL,
+                ]);
+            },
+        });
+
+        await dashboardPage.enterEditMode();
+        await dashboardPage.clickFirstControlSettingsButton();
+    });
+    datalensTest.afterEach(async ({page}: {page: Page}) => {
+        if (skipAfterEach) {
+            return;
+        }
+
+        const dashboardPage = new DashboardPage({page});
+
+        await dashboardPage.deleteDash();
+    });
+
+    datalensTest(
+        '`Apply` button of the group selector applies the selected values',
+        async ({page}: {page: Page}) => {
+            const dashboardPage = new DashboardPage({page});
+
+            await page.locator(slct(DialogGroupControlQa.applyButtonCheckbox)).click();
+
+            await page.locator(slct(ControlQA.dialogControlApplyBtn)).click();
+            await dashboardPage.saveChanges();
+
+            // check that there's no state
+            expect(getUrlStateParam(page)).toEqual(null);
```

Comment:
It's not a good idea to check for absence. If there would be state later this condition would be passed although.